2.0.2/buster/build-time-pip-constraints.txt

```diff
@@ -1,87 +1,88 @@
 # Constraint the version pip will install during docker build
 
 adal==1.2.7
-alembic==1.5.8
+alembic==1.6.5
 amqp==2.6.1
-apache-airflow-providers-amazon==1!1.3.0
+apache-airflow==1!2.0.2+astro.4
```

Comment:
We don't need this as we already have:

https://github.com/astronomer/ap-airflow/blob/7260fe59df278d56d8d95b72f8ae1af84e4e8d92/2.0.2/buster/Dockerfile#L161-L162

Reply:
This has now been removed